content/build/release-notes.md

```diff
@@ -8,6 +8,101 @@ toc_max: 2
+
+The full release note for this release is available
+[on GitHub](https://github.com/docker/buildx/releases/tag/v0.12.0).
+
+### New
+
+- New `--annotation` flag for the `buildx build`, and an `annotations` key in the Bake file, that lets you add OCI Annotations to build results.
+  [#2020](https://github.com/docker/buildx/pull/2020),
+  [#2098](https://github.com/docker/buildx/pull/2098)
+- New experimental debugging features, including a new `debug` command and an interactive debugging console.
+  This feature currently requires setting `BUILDX_EXPERIMENTAL=1`.
+  [#2006](https://github.com/docker/buildx/pull/2006),
+  [#1896](https://github.com/docker/buildx/pull/1896),
+  [#1970](https://github.com/docker/buildx/pull/1970),
+  [#1914](https://github.com/docker/buildx/pull/1914),
+  [#2026](https://github.com/docker/buildx/pull/2026),
+  [#2086](https://github.com/docker/buildx/pull/2086)
+
+### Bug fixes and enhancements
+
+- The special `host-gateway` IP mapping can now be used with the `--add-host` flag during build.
+  [#1894](https://github.com/docker/buildx/pull/1894),
+  [#2083](https://github.com/docker/buildx/pull/2083)
+- Bake now allows adding local source files when building from remote definition.
+  [#1838](https://github.com/docker/buildx/pull/1838)
+- The status of uploading build results to Docker is now shown interactively on progress bar.
+  [#1994](https://github.com/docker/buildx/pull/1994)
+- Error handling has been improved when bootstrapping multi-node build clusters.
+  [#1869](https://github.com/docker/buildx/pull/1869)
+- The `buildx imagetools create` command now allows adding annotation when creating new images in the registry.
+  [#1965](https://github.com/docker/buildx/pull/1965)
+- OpenTelemetry build trace delegation from buildx is now possible with Docker and Remote driver.
+  [#2034](https://github.com/docker/buildx/pull/2034)
+- Bake command now shows all files where the build definition was loaded from on the progress bar.
+  [#2076](https://github.com/docker/buildx/pull/2076)
+- Bake files now allow the same attributes to be defined in multiple definition files.
+  [#1062](https://github.com/docker/buildx/pull/1062)
+- Using the Bake command with a remote definition now allows this definition to use local Dockerfiles.
+  [#2015](https://github.com/docker/buildx/pull/2015)
+- Docker container driver now explicitly sets BuildKit config path to make sure configurations are loaded from same location for both mainline and rootless images.
+  [#2093](https://github.com/docker/buildx/pull/2093)
+- Improve performance of detecting when BuildKit instance has completed booting.
+  [#1934](https://github.com/docker/buildx/pull/1934)
+- Container driver now accepts many new driver options for defining the resource limits for BuildKit container.
+  [#2048](https://github.com/docker/buildx/pull/2048)
+- Inspection commands now show BuildKit worker IDs.
+  [#2063](https://github.com/docker/buildx/pull/2063)
```

Comment:
Remove this one, we don't output worker IDs actually, we just set them. Has been removed from GH release notes.